README.md

```diff
@@ -12,6 +12,7 @@ The most advanced golden test library for Go with **Go 1.24+ generics**, **smart
 - **⚡ High Performance** - Optimized for large files and parallel execution
 - **🔒 Thread Safe** - Perfect for concurrent test execution
 - **📁 Zero Config** - Works out of the box, customizable when needed
+- **🗂️ IDE Integration** - Golden files use `.golden.go` extension for better IDE support
```

Comment:
I wouldn't have listed it here as it's something only used when developing new validators, no?

Reply:
Sorry, I can't get what you said. This section is for describing reason why golden should be used than other similar libraries.

Reply:
I'm sorry for the confusion.

I thought I was reviewing govalid and this addition made no sense to me.

Now, I get it 🤣🤦‍♂️